### PR TITLE
project: Add basic angular nav

### PR DESCRIPTION
**Pull request by hemangsk**

### CODE REVIEW

File: public/javascripts/dashboard.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        }
+    ]
+    $scope.isSelected = function (value) {
+        return value.val == $scope.selected;
+    }
+    $scope.setSelected = function (value) {
+        $scope.selected = value;
+    }
+
+});
```
Comment (hemangsk) [inline public/javascripts/dashboard.js]: Line contains following spacing inconsistencies:
- No newline at EOF.

*Origin: SpaceConsistencyBear, Section: `cli`.*

The issue can be fixed by applying the following patch:

```diff
--- a/public/javascripts/dashboard.js
+++ b/public/javascripts/dashboard.js
@@ -41,4 +41,4 @@
         $scope.selected = value;
     }
 
-});+});
```

Comment (hemangsk) [inline public/javascripts/dashboard.js]: Line contains following spacing inconsistencies:
- No newline at EOF.

*Origin: SpaceConsistencyBear, Section: `cli`.*

The issue can be fixed by applying the following patch:

```diff
--- a/public/javascripts/dashboard.js
+++ b/public/javascripts/dashboard.js
@@ -41,4 +41,4 @@
         $scope.selected = value;
     }
 
-});+});
```

**Review by surender7790 — APPROVED** [review]
Everything seems fine to me.

**Comment by hemangsk** [comment]
@rultor merge

**Pull request merged**